footer.php

```diff
@@ -44,36 +42,36 @@
 					<li><a href="mailto:dj@kbcs.fm">news@kbcs.fm</a></li>
 					<li>425-564-6195</li>
 					</ul>
-				</div><!-- span3 -->
+				</nav><!-- span3 -->
 	
 	
-				<div class="span3">
+				<nav class="span3">
 					<h4>Connect</h4>
 					<ul>
 						<li><a href="<?php echo home_url(); ?>/support/business/">Business Support</a></li>
                         <li><a href="<?php echo home_url(); ?>/donate/">Donate</a></li>
 						<li><a href="<?php echo home_url(); ?>/support/volunteer/">Volunteer</a></li>
-						<li><a href="<?php echo home_url(); ?>/support/">More...</a></li>
+						<li><a href="<?php echo home_url(); ?>/support/">Support Us</a></li>
 					</ul>
 					<h4>Legal</h4>
 					<ul>
                         <li><a href="<?php echo home_url(); ?>/kbcs-public-files/">KBCS Public Files</a></li>
 						<li><a href="https://www.bellevuecollege.edu/trustees/">Controlling Board of KBCS</a></li>
 						<li><a href="https://www.bellevuecollege.edu/events/trustees/">Board Meetings</a></li>
 					</ul>
-				</div><!-- span3 -->
+				</nav><!-- span3 -->
 	
 	
-				<div class="span3">
+				<nav class="span3 bc-footer">
                     <div id="bclogo">
                 	
                 	<p class="bc-service">91.3 KBCS is a public service at</p>
-                    <a href="https://www.bellevuecollege.edu"><img src="<?php bloginfo('stylesheet_directory'); ?>/img/bellevuecollege.png" alt="Bellevue College" /></a></div> <!--bclogo-->
+                    <a role="button" href="https://www.bellevuecollege.edu"><img src="<?php bloginfo('stylesheet_directory'); ?>/img/bellevuecollege.png" alt="Bellevue College" /></a></div> <!--bclogo-->
```

Comment:
This is an inaccurate role here. This is a link, not an action to submit a form or otherwise perform a the type of task a button would perform.

Reply:
Removed this role